### PR TITLE
8113 Feature: Progress and complete icons, additional green colour for notifications

### PR DESCRIPTION
**Pull request by stuartwilkes**
Relates https://github.com/wellcometrust/corporate/issues/8113.

Adds icons and colour to support Org transfer form

### CODE REVIEW

File: src/components/Icon/components/Tick.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { IconSVGProps } from '../Icon';
+
+const SvgTick = (props: IconSVGProps) => (
+  <svg viewBox="0 0 24 24" {...props}>
+    <path
+      id="_3rthkb6hpa"
```
Comment (chriddell) [inline src/components/Icon/components/Tick.tsx]: Think this ID can be removed, it's not used (as far as I can see?)

**Review by chriddell — APPROVED** [review]
Looks fine to me, one minor issue with a superfluous ID

File: src/components/Icon/components/Tick.tsx (hunk 1 above)
Comment (stuartwilkes) [reply src/components/Icon/components/Tick.tsx]: Ah yes - thanks

**Comment by stuartwilkes** [comment]
> Looks fine to me, one minor issue with a superfluous ID

Thanks - removed that id...

**Pull request merged**